invenio_migrator/tasks/users.py

```diff
@@ -56,10 +56,14 @@ def load_user(data):
     if data['note'] == '1':
         confirmed_at = datetime.utcnow()
 
+    hash_prefix = u'invenio-aes'
+    salt = data['password_salt']
+    checksum = data['password']
+    new_password = str.join('$', ['', hash_prefix, salt, checksum, ''])
```

Comment:
we have a mix of old and new password, so it's only if we're dealing with an old password that this is needed